AnalyzerTools/src/AnalyzerParameter.C

```diff
@@ -61,6 +62,7 @@ AnalyzerParameter::AnalyzerParameter(){
   Muon_Tight_ID = "POGTightWithTightIso";
   Muon_Loose_ID = "POGLoose";
   Muon_Veto_ID = "POGLoose";
+  Muon_RECO_SF_Key = "";
```

Comment:
why is there no default string set for Muon_RECO_SF_Key in line 65?

Reply:
well there isn't a case of user us default values of this class..